app/views/handlers/question.py

```diff
@@ -211,5 +211,7 @@ def clear_radio_answers(self):
                 answer_ids_to_remove.append(answer["id"])
 
         if answer_ids_to_remove:
-            self.questionnaire_store_updater.remove_answers(answer_ids_to_remove)
+            self.questionnaire_store_updater.remove_answers(
+                answer_ids_to_remove, self.current_location.list_item_id
+            )
```

Comment:
Just realized, questionnaire store updater already has current_location as an attribute, so we don't need to pass it in this case. We do `list_item_id = list_item_id or self._current_location.list_item_id` in other places to make it flexible.